src/hist.jl

```diff
@@ -18,9 +18,9 @@ Plots.@deps density path
 # ---------------------------------------------------------------------------
 # cumulative density
 
-@recipe function f(::Type{Val{:cdensity}}, x, y, z; trim=false,
-                   npoints = 200)
```

Comment:
why change `npoints` to `n` here? And why add the type assertion? IMHO this change is unnecessary.